### ISSUE TITLE
How to get Nextor Working

### ISSUE THREAD

**Issue by naeloob** [MiSTer-devel/MSX_MiSTer#13]
Nextor is an advanced bios offered by KDL.
Still not tested on secondary SD card, but i cannot make it works on VHD file, and i have not seen anyone report it works in any way.

Can you make it work?

Regards

**Comment by MP2E** [comment]
I will look into that, been a little distracted by the NES and Genesis cores lately, but Nextor running would be nice.

**Comment by ducasp** [comment]
Create a SDBIOS in your SD-CARD, the BIOS that goes with the firmware doesn't have Nextor Kernel. I've made a video that explains how to do it for SM-X/Zemmix Neo/OCM, my guess is that it might work for you as well...

https://youtu.be/h2RQ-0zdfQ8

The audio is in Portuguese but there are english subtitles available

**Comment by tomxp411** [comment]
I was finally able to get it to work, but it required updated NEXTOR BIOS. 

https://github.com/Konamiman/Nextor/releases

Grab **Nextor-2.1.0.MegaFlashSDSCC.2-slots.ROM** and **NEXTOR.SYS**.

- Rename the ROM file to OCM-BIOS.DAT and replace the existing one on your SD card or VHD. 
- Copy NEXTOR.SYS to the SD card, overwriting the one that's already there. 
- Reboot your MSX core from the SD card or the VHD. 

On boot, you should see:

`NEXTOR.SYS version 2.11
Copyright (2019) Konamiman`

followed by your usual AUTOEXEC boot messages.

To confirm this is working, try the **drivers** command.
It should say 
`MegaFlashROM SCC+ SD v.1.3 on slot 3-2.
Type: device-based
Drives assigned at boot time: 2, from A: to B:`

**Comment by sorgelig** [comment]
Long MSX reset is hard reset with bios loading.

**Comment by tomxp411** [comment]
Yes, but it doesn’t seem, to work consistently, especially if the initial boot failed.

**Comment by Vampier** [comment]
> I was finally able to get it to work, but it required updated NEXTOR BIOS.
> 
> https://github.com/Konamiman/Nextor/releases [snip]

tompx411 this is indeed the way to go - try Nextor 2.10 instead - it works fine for me.
